SelfReflection/SelfReflection/Assets/Scripts/Pressure Plate/PressurePlate.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PressurePlate : MonoBehaviour
+{
+    public Rigidbody wall;
+
+    private void OnTriggerEnter(Collider other)
+    {
+        gameObject.transform.position = new Vector3(10, (float)-0.24, -10);
```

Comment:
Shouldn't hard code position something like this is better practice
gameObject.transform.position = new Vector3(gameObject.transform.position.x, gameObject.transform.position.y - sinkAmount, gameObject.transform.position.z);
where sinkAmount is a public variable or something

Reply:
same for OnTriggerExit